packages/recomponents/src/components/r-pagination/r-pagination.scss

```diff
@@ -0,0 +1,17 @@
+.r-pagination {
+    text-align: center;
+}
+
+.r-pagination-numbers {
+    margin: var(--pagination-buttons-margin);
+}
+
+.r-pagination-description {
+    margin-left: 0;
+}
+
+.r-pagination-input {
+    margin-right: 0;
+    display: flex;
+    width: var(--pagination-input-width)
+}
```

Comment:
This is duplicated between `r-pagination-control.scss` and r-pagination.scss`, can we remove this?

![Proof](https://i.paste.pics/34d00c8c97e51fca421bb11a4a167c83.png)